infrastructure/cdn-in-a-box/optional/docker-compose.debugging.yml

```diff
@@ -71,25 +62,12 @@ services:
       args:
         TRAFFIC_MONITOR_RPM: traffic_monitor/traffic_monitor.rpm
     command: /usr/bin/true
-  # The trafficops-go-nondebug service ensures that the trafficops-go base image
-  # exists before trying to build trafficops-go-debug.
-  trafficops-go-nondebug:
-    image: trafficops-go
+  # The trafficops-nondebug service ensures that the trafficops base image
+  # exists before trying to build trafficops-debug.
+  trafficops-nondebug:
+    image: trafficops
     build:
-      context: .
-      dockerfile: traffic_ops/Dockerfile-go
```

Comment:
It's still not finding the Dockerfile because the Dockerfile doesn't exist in the directory of the build context. Its path needs to be specified.

```
[user@computer cdn-in-a-box]$ docker-compose -f docker-compose.yml -f optional/docker-compose.debugging.yml build trafficops-nondebug
Building with native build. Learn about native build in Compose here: https://docs.docker.com/go/compose-native-build/
Building trafficops-nondebug
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/user/go/src/github.com/apache/trafficcontrol/Dockerfile: no such file or directory
```

Reply:
I could have sworn this was working... maybe it was lost in the rebase.